packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -2182,7 +2182,7 @@ function pushLink(
           precedenceSet.add(resource);
         }
         if (renderState.boundaryResources) {
-          renderState.boundaryResources.add(resource);
+          renderState.boundaryResources.add(resource); // TODO
```

Comment:
@gnoff Do we need to be able to transfer this to the boundary? We might not need to track them across resuming if assume that they're always flushed in the shell. We might need to keep track of them during a single render pass. Like if we the same resource is discovered in a different boundary which then gets completed first.

Reply:
We don't need to track across resuming b/c all style resources should flush with the shell. But within the resume render we still need to track this association b/c the first boundary to complete that depends on this resource needs to create it as part of the reveal instruction and that could be a different boundary than the one that first observed it. (I think this is what you are saying but not sure so I'm restating it)